### ISSUE TITLE
Default font's spacing on mac incorrect

### ISSUE THREAD

**Issue by sbrugman** [parrt/tensor-sensor#34]
![Default font](https://user-images.githubusercontent.com/9756388/134520859-5fa2ed1c-3896-42e4-a997-564ced745048.png)

Compare with _courier new_

![Courier New](https://user-images.githubusercontent.com/9756388/134521075-11431ac0-7953-4614-944f-b345fea5cf3b.png)

Relevant code:
```python
with ts.explain(fontname="Courier New"):
    y = np.multiply.outer(x, np.arange(2.0))
```

**Comment by parrt** [comment]
Weird. is Consolas font installed on your system? Is it Intel or M1?

**Comment by sbrugman** [comment]
Consolas wasn't installed in my environment (Intel). The fallback font is "Dejavu Sans", which fails to display the code as seen above. 

Advising users to install the font may involve recreating the environment etc. An alternative solution to consider is to explicitly specify a fallback font. This is much more user friendly in my opinion at nearly no burden on the package. For instance, "DejaVu Sans Mono" works well. This can be trivially achieved providing a sequence of font names:

<img sc="https://user-images.githubusercontent.com/9756388/134952084-df92eb01-5bd8-4bcc-8f7f-f603f10eb067.png">

**Comment by parrt** [comment]
agreed this is a good change. I guess your PR only sets the arg default, but there's no code to fall back to the indicated font. Font name is now a tuple by default, which is really a comment for the developer.  Shouldn't there be logic to check for font installations and use one that works?

**Comment by sbrugman** [comment]
Matplotlib handles all that logic:

> WARNING:matplotlib.font_manager:findfont: Font family ('Consolas', 'Consolas Sans Mono') not found. Falling back to DejaVu Sans.

**Comment by parrt** [comment]
You are kidding me!? Wow. that's insane. You just took me to school!

**Comment by parrt** [comment]
By the way gotta use "Fixes #34" not "Fix for #34" to get autoclose. :)
